app/models/metric/helper.rb

```diff
@@ -198,4 +198,12 @@ def self.get_time_interval(obj, timestamp)
 
     start_time..timestamp
   end
+
+  def self.latest_metrics(resource_type, resource_ids = nil)
+    metrics = Metric.where(:resource_type => resource_type)
+    metrics = metrics.where(:resource_id => resource_ids) if resource_ids
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.where('timestamp > ?', 10.minutes.ago.utc)
```

Comment:
@simon3z That is the time interval we agreed on for "realtime" metrics that we display in the dashboard. Same value is used here: 
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/services/container_dashboard_service.rb#L395 
(we can change it easily of course)